src/Control/Coroutine.purs

```diff
@@ -74,8 +91,9 @@ fuseWith zap fs gs = freeT \_ -> go (Tuple fs gs)
   go :: Tuple (Co f m a) (Co g m a) -> m (Either a (h (Co h m a)))
   go (Tuple fs' gs') = do
     next <- sequential
-              (lift2 (zap Tuple) <$> parallel (resume fs')
-                                 <*> parallel (resume gs'))
+      ( lift2 (zap Tuple) <$> parallel (resume fs')
+          <*> parallel (resume gs')
+      )
```

Comment:
```suggestion
      ( lift2 (zap Tuple)
          <$> parallel (resume fs')
          <*> parallel (resume gs')
      )
```

Reply:
Yeah, your suggestion is better than what it did.